LoopKitUI/CGMManagerUI.swift

```diff
@@ -15,8 +15,8 @@ public protocol CGMManagerUI: CGMManager, DeviceManagerUI {
     /// If this method returns nil, it's expected that `init?(rawState: [:])` creates a non-nil manager
     static func setupViewController(glucoseTintColor: Color, guidanceColors: GuidanceColors) -> (UIViewController & CGMManagerSetupViewController & CompletionNotifying)?
```

Comment:
[nit] Any need to be notified of glucose units changes during setup? (I suspect so.)

Reply:
I was going to ass this as well, but as you can see, the glucose units are not passed to the setup view controller. So currently there isn't a use case for this.

Reply:
I think as part of the general protocol of CGM setup/settings it would need to be there, but I suppose we can defer until later.